docs/src/components/Layout.js

```diff
@@ -189,14 +221,26 @@ const Layout = ({
     h1: (props) => <Heading size='xxl' {...props} />,
     h2: (props) => <Heading size='xl' {...props} />,
     h3: (props) => <Heading size='l' {...props} />,
-    h4: (props) => <Heading size='m' {...props} />,
```

Comment:
Same functionality should be implemented for other headings (H1, H2, etc) as well